lib/resourceapply/security.go

```diff
@@ -28,7 +81,12 @@ func ApplySecurityContextConstraintsv1(ctx context.Context, client securityclien
 		return nil, false, nil
 	}
 
-	reconcile := resourcemerge.EnsureSecurityContextConstraints(*existing, *required)
+	reconcile, requiredMerges := resourcemerge.EnsureSecurityContextConstraints(*existing, *required)
+
+	if reconciling {
+		trackModifiedScc(required.Name, requiredMerges)
```

Comment:
I think we can do this regardless of whether we're in `reconciling` mode, but :shrug: also not all that likely that folks turn straight from an in-progress *-to-4.13 update and target 4.14.